libsql-server/src/connection/libsql.rs

```diff
@@ -164,10 +166,54 @@ impl<T> std::fmt::Debug for LibSqlConnection<T> {
     }
 }
 
+#[derive(Clone, Copy)]
+pub struct InhibitCheckpointWalWrapper;
+
+impl<W: Wal> WrapWal<W> for InhibitCheckpointWalWrapper {
+    fn checkpoint<B: libsql_sys::wal::BusyHandler>(
+        &mut self,
+        _wrapped: &mut W,
+        _db: &mut libsql_sys::wal::Sqlite3Db,
+        _mode: libsql_sys::wal::CheckpointMode,
+        _busy_handler: Option<&mut B>,
+        _sync_flags: u32,
+        _buf: &mut [u8],
+    ) -> libsql_sys::wal::Result<(u32, u32)> {
```

Comment:
let's put a log message here, debug level maybe? So that we can track if some connections get inhibited often

Reply:
sure